src/common/config/constants.ts

```diff
@@ -285,21 +285,14 @@ export const OPERATOR_NAME_CNV = 'cnv';
 export const OPERATOR_NAME_LSO = 'lso';
 export const OPERATOR_NAME_ODF = 'odf';
 export const OPERATOR_NAME_LVM = 'lvm';
-export const OPERATOR_NAME_CVO = 'cvo';
```

Comment:
I removed the ones (with the corresponding translation stuff) that were unused in the codebase.